src/jit/optimizer.cpp

```diff
@@ -7837,7 +7837,9 @@ GenTree *           Compiler::optIsBoolCond(GenTree *   condBranch,
     if  (opr2->gtOper != GT_CNS_INT)
         return  NULL;
 
```

Comment:
Since you are editing around this.. Coding Convention (CC): replace NULL with nullptr